operator/config/serverconfigs/mlserver.yaml

```diff
@@ -96,6 +96,8 @@ spec:
           resourceFieldRef:
             containerName: mlserver
             resource: requests.memory
+      - name: SELDON_USE_SERVER_WITH_DEPLOYMENT
```

Comment:
For consistency, if you decide to make the suggested name change, those should probably be `SELDON_USE_DEPLOYMENTS_FOR_SERVERS`